src/creatures/players/player.cpp

```diff
@@ -243,9 +243,12 @@ Item* Player::getWeapon(Slots_t slot, bool ignoreAmmo) const
       bool found = false;
       for (Item* ammoItem : container->getItemList()) {
         if (ammoItem->getAmmoType() == it.ammoType) {
-          item = ammoItem;
-          found = true;
-          break;
+			const ItemType &ammoItemType = Item::items[ammoItem->getID()];
+			if (ammoItemType.minReqLevel != 0 && level >= ammoItemType.minReqLevel) {
```

Comment:
Do we need to check if ammoItemType.minReqLevel != 0?
player level is always > 0, I think the second condition is enough, isn't it?
also, I think if you define the minReqLevel for that arrow as 0 and there are only this type of arrows inside the quiver, "found" will never be true (the if condition won't be satisfied because ammoItemType.minReqLevel == 0).